CMCL-0000: Add user manual page for SimplePlayerController sample code (#1004)
Unity-Technologies/com.unity.cinemachine: com.unity.cinemachine/Documentation~/SimplePlayerController.md

| **AimTarget Manager** | Target to Aim towards. If null, the aim is defined by the forward vector of this gameObject |
| **Fire Event** | Event that's triggered when firing |

### SimplePlayerOnSurface

This behaviour keeps a player upright on surfaces.  It can be used to make the player walk on walls and ceilings or on the surfaces of arbitrary meshes.  It rotates the player so that its Up direction matches the surface normal where it is standing.  This script assumes that the pivot point of the player is at the bottom.

You can see an example of its usage in the _FreeLook on Spherical Surface_ sample scene.

Raycasts are used to detect walkable surfaces.

When using this component, SimpleSplayerController's Up Mode should be set to _Player_, and it should not have a Character Controller component, as that does not play nicely with nonstandard Up directions.

Also, when CinemachineCameras are being used to track the character, the [CinemachineBrain](CinemachineBrain.md)'s World Up Override setting should be set to the Player, so that the Camera's Up will match the Player's Up.

**Settings:**

| Setting | Description |
| :--- | :--- |

> | **Rotation Damping** | How fast the player rotates to match the surface normal |
> | **Ground Layers** | What layers to consider as ground |
> | **Max Raycast Distance** | How far to raycast when checking for ground |
> | **Player Height** | The approximate height of the player.  Used to compute where raycasts begin |
> | **Free Fall Recovery** | If enabled, then player will fall towards the nearest surface when in free fall |
> | **Surface Changed** | This event is fired when the player moves from one surface to another.  If the surfaces are moving, then this is a good opportunity to reparent the player |

Reviewer: ```suggestion
| **Rotation Damping** | How fast the player rotates to match the surface normal. |
| **Ground Layers** | Layers to consider as ground. |
| **Max Raycast Distance** | How far to raycast when checking for ground. |
| **Player Height** | The approximate height of the player.  Used to compute where raycasts begin. |
| **Free Fall Recovery** | Makes the player fall towards the nearest surface when in free fall. |
| **Surface Changed** | This event is fired when the player moves from one surface to another.  If the surfaces are moving, then this is a good opportunity to reparent the player. |
```
